hawkular-wildfly-agent-ansible-playbook/roles/fetch-and-unpack-wildfly/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This only works with Ansible 2.0 (because unarchive src is an http URL)
+---
+  - name: Create WildFly installation directory
+    file: path="{{wildfly_install_dir}}" state=directory mode=0755
+
+  - name: Download and unpack WildFly app server
+    unarchive:
```

Comment:
This always pulls from the remote repo no matter on how many hosts we install.
I had a playbook for pulling that into the ansible "server" and then use that url. See https://github.com/pilhuhn/hawkular-agent-ansible/blob/master/roles/fetch_wildfly/tasks/main.yml